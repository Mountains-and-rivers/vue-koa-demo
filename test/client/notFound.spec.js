import { shallowMount } from '@vue/test-utils'
import NotFound from '@/components/NotFound'

describe('NotFound.vue', () => {
  let wrapper
  beforeEach(() => {
    wrapper = shallowMount(NotFound)
  })

  test('render text 404', () => {
    expect(wrapper.html()).toContain('<h1><strong>404</strong> 啊哈页面不见了</h1>')
  })

  test('notFound.vue snapshot test', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
