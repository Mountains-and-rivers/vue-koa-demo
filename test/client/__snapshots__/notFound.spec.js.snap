// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound.vue notFound.vue snapshot test 1`] = `
<div
  class="notfound"
>
  <h1>
    <strong>
      404
    </strong>
     啊哈页面不见了
  </h1>
</div>
`;
